platform/vendor/spire/chart/templates/spiffe-csi-driver.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: storage.k8s.io/v1
```

Comment:
maybe we can add the same condition over here:

```yaml
{{- if .Values.csi.enabled }}
...
{{- end }}
```

Reply:
Good catch, done!